apps/server/src/schemas/apiKey.schema.ts

```diff
@@ -0,0 +1,16 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+import { Recipient, RecipientSchema } from './recipient.schema';
```

Comment:
i think don't need this import below:
`'import { Recipient, RecipientSchema } from './recipient.schema';`